src/components/AttendeeForm.tsx

```diff
@@ -0,0 +1,225 @@
+    const files = event.target.files
+    if (files !== null && files.length > 0) {
+      const selected = files[0]
+      setInputs({ ...inputs, avatar: selected })
+    }
+  }
+
+  const showErrorMsg = (key: keyof Inputs): boolean =>
+    errors[key] !== undefined && touched[key] !== undefined
+
+  const handleSubmit = () => (event: React.MouseEvent<HTMLButtonElement, MouseEvent>) => {
+    event.preventDefault()
+
+    let obj = {}
+    for (const cle in inputs) {
+      obj = { ...obj, [cle]: true }
+    }
+
+    setTouched(obj)
+
+    if (Object.keys(errors).length === 0) {
+      if (user !== null) {
+        const data = {
+          ...inputs,
+          isCaptain: false,
+          tournament: tournamentId,
+          user: user.id,
+        }
+
+        pb.collection('attendees')
+          .create(data)
+          .then(() => onSuscribeOk())
+          .catch(() => setSubmitError('An unkwoned error appened'))
+      }
+    }
+  }
+
+  return (
+    <form
+      className="flex w-[26rem] flex-col gap-3 rounded-lg border-2 border-gray-400 bg-white p-4"
+      noValidate={true}
+    >
+      <Input
+        id="riotId"
+        label="Riot ID"
+        type="text"
+        placeholder="summonerName#TAG"
+        onChange={handleChange('riotId')}
+        onBlur={handleBlur('riotId')}
+        errorMsg={errors.riotId ?? ''}
+        showErrorMsg={showErrorMsg('riotId')}
+      />
+      {
+        <SelectInput
+          id="currentElo"
+          label="Current Elo"
+          value={inputs['currentElo']}
+          values={LolElo.values}
+          valuesLabels={LolElo.values.map(v => LolElo.label[v])}
+          onChange={handleSelectChange('currentElo')}
+          errorMsg={errors.currentElo ?? ''}
+          showErrorMsg={showErrorMsg('currentElo')}
+        />
+      }
+      <Input
+        id="comment"
+        label="Quelque chose à préciser sur cet elo minable ?"
+        type="text"
+        placeholder="c'est à cause de mes mates !"
+        onChange={handleChange('comment')}
+        onBlur={handleBlur('comment')}
+        errorMsg={errors.comment ?? ''}
+        showErrorMsg={showErrorMsg('comment')}
+      />
+
+      {
+        <SelectInput
+          id="role"
+          label="Role"
+          value={inputs['role']}
+          values={TeamRole.values}
+          valuesLabels={TeamRole.values.map(v => TeamRole.label[v])}
+          onChange={handleSelectChange('role')}
+          errorMsg={errors.role ?? ''}
+          showErrorMsg={showErrorMsg('role')}
+        />
+      }
+      {
+        <SelectInput
+          id="championPool"
+          label="Champion Pool"
+          value={inputs['championPool']}
+          values={['One trick poney', 'Limité', 'Modeste', 'Pas mal', 'Ça fait beaucoup la non']}
+          valuesLabels={[
+            'One trick poney',
+            'Limité',
+            'Modeste',
+            'Pas mal',
+            'Ça fait beaucoup la non',
```

Comment:
```suggestion
            'Ça fait beaucoup là non',
```

De plus, peut-être pas en faire un `select` en base, mais en faire une union en TS ?

Reply:
Une union TS pour cette piscine de champions ?